lib/msf/ui/tip.rb

```diff
@@ -33,6 +33,8 @@ def self.highlight(string)
         "Metasploit can be configured at startup, see #{highlight('msfconsole --help')} to learn more",
         "Display the Framework log using the #{highlight('log')} command, learn more with #{highlight('help log')}",
         "Adapter names can be used for IP params #{highlight('set LHOST eth0')}",
+        "Use #{highlight('sessions -1')} to interact with the last opened session",
```

Comment:
Details on the implementation behind this one - when passing only a number to the session, it defaults to interact mode:

https://github.com/rapid7/metasploit-framework/blob/2a6a8e4c23aa57cbda5a93ec9229fd9ac43112e6/lib/msf/ui/console/command_dispatcher/core.rb#L1150-L1153

The session manager is then hardcoded to support the -1 value to get the highest session id:

https://github.com/rapid7/metasploit-framework/blob/2a6a8e4c23aa57cbda5a93ec9229fd9ac43112e6/lib/msf/core/session_manager.rb#L277-L280

Reply:
Looks good to me, thanks for clarifying this @adfoster-r7, I'll go ahead and merge this so long.